addons/mail/views/res_config_settings_views.xml

```diff
@@ -135,6 +135,33 @@
                         </div>
                     </div>
                 </div>
+                <div id="companies_setting" position="inside">
+                    <br/>
+                    <div class="o_setting_right_pane" id="mail_templates_setting"
+                        groups="mail.group_mail_template_editor,base.group_system">
+                        <span class="o_form_label">Email Templates</span>
+                        <div class="text-muted">
+                            Customize the look and feel of automated emails
+                        </div>
+                        <div class="d-flex flex-row align-items-center">
+                            <span class="o_settings_color_picker">Header Color</span>
+                            <field name="primary_color" widget="color"/>
+                        </div>
+                        <div class="d-flex flex-row align-items-center">
+                            <span class="o_settings_color_picker mt-1">Button Color</span>
+                            <field name="secondary_color" widget="color"/>
+                        </div>
+                        <button name="open_mail_notification_paynow" icon="fa-arrow-right"
+                            type="object" string="Update Order/Invoice Header/Footer"
```

Comment:
This is incorrect, as this layout is used in a lot of cases (and will be used even more soon). Moreover we should not talk about order / invoice here, as this should be a bit more agnostic.